ui/pages/onboarding-flow/privacy-settings/privacy-settings.js

```diff
@@ -376,29 +393,105 @@ export default function PrivacySettings() {
-                    </div>
-                  ) : (
-                    <ButtonSecondary
-                      size={ButtonSecondarySize.Lg}
-                      onClick={(e) => {
-                        e.preventDefault();
-                        dispatch(showModal({ name: 'ONBOARDING_ADD_NETWORK' }));
-                      }}
+                {window.metamaskFeatureFlags?.networkMenuRedesign ? (
+                  <Box paddingTop={4}>
+                    <Box
+                      display={Display.Flex}
+                      flexDirection={FlexDirection.Column}
+                      gap={5}
                     >
-                      {t('onboardingAdvancedPrivacyNetworkButton')}
-                    </ButtonSecondary>
-                  )}
-                </Box>
+                      {[CHAIN_IDS.MAINNET, CHAIN_IDS.LINEA_MAINNET].map(
+                        (chainId) => (
+                          <Box
+                            key={chainId}
+                            className="privacy-settings__customizable-network"
+                            onClick={() =>
+                              console.log(`chain ${chainId} clicked`)
+                            }
+                            display={Display.Flex}
+                            alignItems={AlignItems.center}
+                            justifyContent={JustifyContent.spaceBetween}
+                          >
+                            <Box
+                              display={Display.Flex}
+                              alignItems={AlignItems.center}
+                            >
+                              <AvatarNetwork
+                                src={CHAIN_ID_TO_NETWORK_IMAGE_URL_MAP[chainId]}
+                              />
+                              <Box textAlign={TextAlign.Left} marginLeft={3}>
+                                <Text variant={TextVariant.bodySmMedium}>
+                                  {NETWORK_TO_NAME_MAP[chainId]}
+                                </Text>
+                                <Text
+                                  variant={TextVariant.bodyXs}
+                                  color={TextColor.textAlternative}
+                                >
+                                  {(function () {
+                                    const url = allNetworks.find(
+                                      (network) => network.chainId === chainId,
+                                    )?.rpcUrl;
+
+                                    // Get just the domain, not the infura key
+                                    return url?.slice(
+                                      0,
+                                      url.indexOf('/', url.indexOf('://') + 3),
+                                    );
+                                  })()}
+                                </Text>
+                              </Box>
+                            </Box>
+                            <ButtonIcon
```

Comment:
This can just be `Icon`

Reply:
I did this for accessibility RE: other comment